pypika/clickhouse/condition.py

```diff
@@ -0,0 +1,42 @@
+from pypika.terms import Function
+
+
+class If(Function):
+    def __init__(self, condition, then, else_, alias: str = None, schema: str = None):
+        self._else_ = else_
+        self._then = then
+        self._condition = condition
+        self.alias = alias
+        self.schema = schema
+        self.args = ()
+        self.name = 'if'
+
+    def get_sql(self, with_alias=False, with_namespace=False, quote_char=None, dialect=None,
+                **kwargs):
+        return '{name}({condition},{then},{else_}){alias}'.format(
+            name=self.name,
+            condition=self._condition,
+            then=self._then,
+            else_=self._else_,
+            alias=' ' + self.alias if self.alias else ''
+        )
+
+
+class MultiIf(Function):
+    def __init__(self, conditions: tuple, alias: str = None, schema: str = None):
+        """
+        :param conditions: condition1, then1, condition2, then2...else
+        """
+        self._conditions = conditions
+        self.alias = alias
+        self.schema = schema
+        self.args = ()
```

Comment:
Doesn't look like args or schema are used again?

Reply:
This is really don't need. But I remember that I got in some case(earlier):

> AttributeError: 'MyCustomFunc' object has no attribute args

At the moment all works fine. Thank you.